README.md

```diff
@@ -140,34 +141,34 @@ _Note: Attribute keys starting with `ATL.` require the [Active Token Effects](ht
+| `Devil's Sight`             | _Warlock invocation_ / _Monster feat_ | `Teufelssicht`                 | `Vision de diable` / `Vision du diable` / `Vue de diable` / `Vue du diable` | `Vista del diablo`                | `Visão diabólica` / `Visão do diabo`              |
+| `Divine Sense`              | _Paladin feat_                        | `Göttliches Gespür`            | `Perception divine`                                                         | `Sentidos divinos`                | `Sentido divino`                                  |
+| `Echolocation`              | _Monster feat_                        | `Echolot`                      | `Écholocation` / `Écholocalisation`                                         | `Ecolocalización`                 | `Ecolocalização`                                  |
+| `Eldritch Adept`            | _Feat_                                | `Schauerlicher Adept`          | `Adepte occulte`                                                            | `Adepto sobrenatural`             | `Adepto místico`                                  |
+| `Eldritch Invocation`       | _Warlock feat_                        | `Schauerliche Anrufung`        | `Invocation occulte` / `Manifestation occulte`                              | `Invocación sobrenatural`         | `Invocação mística`                               |
+| `Ethereal Sight`            | _Monster feat_                        | `Ätherische Sicht`             | `Vision éthérée` / `Vue éthérée`                                            | `Visión etérea`                   | `Visão etérea`                                    |
+| `Fairy`                     | _Race_                                | `Fee`                          | `Fée`                                                                       | `Hada`                            | `Fada`                                            |
+| `Ghostly Gaze`              | _Warlock invocation_                  | `Geisterhafter Blick`          | `Regard fantomatique`                                                       | `Mirada fantasmal`                | `Olhar fantasmagórico`                            |
+| `Hexblood`                  | _Race_                                | `Hexblut`                      | `Sang maudit`                                                               | `Sangre maléfica`                 | `Sangre maléfica`                                 |
+| `Hollow One`                | _Supernatural Gift_                   | `Leerwandler`                  | `Celui-qui-est-creux`                                                       | `Aquel que está vacío`            | `Oco`                                             |
+| `Magic Awareness`           | _Barbarian (Path of Wild Magic) feat_ | `Magische Wahrnehmung`         | `Conscience magique`                                                        | `Conciencia mágica`               | `Percepção mágica` / `Vestígios mágicos`          |
```

Comment:
Perhaps this translation needs confirmation as I took it from the weird sources.

Reply:
"Hexblood" is "Sangue Maldito".